mypy/types.py

```diff
@@ -662,6 +662,8 @@ class CallableType(FunctionLike):
                  'min_args',  # Minimum number of arguments; derived from arg_kinds
                  'is_var_arg',  # Is it a varargs function?  Derived from arg_kinds
                  'is_kw_arg',  # Is it a **kwargs function?  Derived from arg_kinds
```

Comment:
Do we need both `is_kw_arg` and `kw_arg` etc. Maybe can make `is_kw_ars` a property that returns `kw_arg is not None`? `CallableType` has already loads of fields.